tools/enso4igv/pom.xml

```diff
@@ -80,10 +80,9 @@
                 <configuration>
                     <compilerId>frgaal</compilerId>
                     <source>21</source>
-                    <target>1.8</target>
+                    <target>17</target>
```

Comment:
We don't control the JDK VSCode is running on. Hence, I'd suggest to support still JDK 11.  If not continue with `1.8` - I don't understand why there should be a need to upgrade to 17?

Reply:
Let's [leave the resolution](https://github.com/enso-org/enso/pull/12855#issuecomment-2820048420) for later.